index.html

```diff
@@ -5,6 +5,7 @@
 <body>
 <h1>Hello World</h1>
 <h1> Hi Nancy</h1>
+<h1> I make a small change</h1>
 
```

Comment:
nice one!

Reply:
so do you actually have a thing to accept it.